definitions/divorce/json/CaseEventToFields.json

```diff
@@ -6394,7 +6438,9 @@
     "PageFieldDisplayOrder": 2,
     "DisplayContext": "OPTIONAL",
     "PageID": "bulkcasehearing",
-    "PageLabel": "Bulk Case Court Hearing Details"
+    "PageLabel": "Bulk Case Court Hearing Details",
+    "CallBackURLMidEvent": "${CCD_DEF_COS_URL}/sol-dn-review-petition",
+    "RetriesTimeoutURLMidEvent": "120,120"
```

Comment:
How come those are moving again?

Reply:
Good question, not sure tbh.. wasn't me that's for sure. Maybe it's the Excel to JSON parser

Reply:
Fixed it @hosainnet 👍